clientd3d/statgame.c

```diff
@@ -375,14 +389,15 @@ void GameIdle(void)
    AnimationTimerProc(hMain, 0);
    HandleKeys();
 
-   // Are we the active foreground application?
    AnimationTimerAbort();
+   BackgroundSleep();
 }
 /****************************************************************************/
 void GameEnterIdle(HWND hwnd, UINT source, HWND hwndSource)
 {
-   // When menu or dialog goes up, start animation timer in the background
-   AnimationTimerStart();
+   // When menu or dialog goes up during gameplay, start animation timer in the background
+   if (game_state == GAME_PLAY)
```

Comment:
With this change, does the two knights bonking animation in the About dialog still work when the client is not yet logged in?  Maybe not a huge deal if not but it would be good to know.